packages/design-system-docs/src/scripts/components/UsaBanner.jsx

```diff
@@ -4,23 +4,23 @@ import path from 'path';
 
 const UsaBanner = function (props) {
   const bannerHeader = props.isBannerOpen
```

Comment:
This file should be removed altogether, the doc site should just be using the core USABanner moving forward